Chapters/2-5-Chapter12.md

```diff
@@ -0,0 +1,175 @@
+## Chapter 12
+
+"Are you sure they're landing here?" asked Kipnu. "I think we came in from over there."
+
+"I'm sure of it. Nanta decides which way to turn depending on the currents. She's not too fond of her job, but she keeps the boat running smoothly."
+
+"Sounds like you know her well."
+
+"I voluntarily traveled to Ulukoma... Tends to raise a few eyebrows."
+
+The three had taken a kelping canoe and paddled out to the space where the ships came in, waiting for their moment. Kipnu looked out at the dark sea and the hint of red and infrared that lie above it. There was far more he could see now that his eyes had adjusted to the dark. He wondered at Casra's stoic nature, how thi held the same expression no matter the hardship: calm, intelligent, and resolute. However this plan turned out, he was certain that a better one didn't exist.
```

Comment:
> Kipnu looked out at the dark sea and the hint of red and infrared that lie above it

honey, infrared is invisible. What's the visible spectrum of these people? I want exact frequencies!!!

Reply:
> He wondered at Casra's stoic nature,

Something sounds off here. I think it's "wondered at"... did you mean "wondered about"?
Also if suffice to say `He wondered how Casra held the same expression no matter the hardship. Thi stoicism made her calm, intelligent, resolute.`